setup.py

```diff
@@ -46,6 +46,8 @@
     ],
     keywords="functions-framework",
     packages=find_packages(where="src"),
+    package_data={"functions_framework": ["py.typed"]},
+    zip_safe=False,
```

Comment:
This is needed per https://mypy.readthedocs.io/en/stable/installed_packages.html#creating-pep-561-compatible-packages

Reply:
Good find, but I'm now seeing that this recommendation has been removed. https://github.com/python/mypy/issues/8802#issuecomment-1676461805.

Reply:
You are right, I removed the flag :+1: